test/mock_compatibility.js

```diff
@@ -0,0 +1,87 @@
+// For the convenience of unit testing, add these to the global namespace
```

Comment:
`mock_compatibility` doesn't sound like a right file name. If you move out the global constants & functions tests elsewhere, then the remaining tests here are mostly about parameter validation.